README.md

```diff
@@ -16,6 +16,7 @@ Response:
   id: String,
   cid: String,
   providerAddress: String,
+  peerID: String,
```

Comment:
Let's call this `providerPeerID` to make it less ambiguous and more consistent with `providerAddress`. 

Other SPARK components may get their peer IDs in the future, too. At that point, it will become difficult to tell which peer is the `peerID` belonging to.

Also: have you considered encoding the peerID into the `providerAddress`? Both options have different pros and cons. I am OK with adding another field, as you are doing now.

Reply:
> Other SPARK components may get their peer IDs in the future, too. At that point, it will become difficult to tell which peer is the peerID belonging to.


E.g. each Station module instance already has a peer ID used for libp2p connections. See [`Zinnia.peerId`](https://github.com/filecoin-station/zinnia/blob/main/docs/building-modules.md#zinniapeerid)